docs: Add instructions for accessing the editor element in js (#53)
Infomaniak/android-rich-html-editor: README.md

Spellcheck is activated by default but if you don't want to have any spellcheck done on the editor content,
you can do that by doing so.

### Acces the editor element inside the DOM

> If you need to access the editor inside some custom javascript code, you can call the globally avaialble `getEditor()` method.

Reviewer: ```suggestion
If you need to access the editor within custom javascript code, you can call the globally available `getEditor()` method.
```
